src/com/lftechnology/java/training/srijan/basics/generics/GenericsInheritance.java

```diff
@@ -0,0 +1,24 @@
+package com.lftechnology.java.training.srijan.basics.generics;
+
+import java.util.logging.Logger;
+public class GenericsInheritance {
+	private static Logger LOGGER = Logger.getLogger(GenericsInheritance.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)